### PR TITLE
Coursera PyTorch: primer + Week 4 + Week 5 REINFORCE + Week 6 Seq2seq

### PR DESCRIPTION
**Pull request by yhn112**
Partially addresses #343 (primer + all of week 4 + week 5 REINFORCE).

### CODE REVIEW

File: week4_approx/dqn_atari_pytorch.ipynb [hunk 1]
```diff
@@ -0,0 +1,1534 @@
+{
```
Comment (dniku) [inline week4_approx/dqn_atari_pytorch.ipynb:1]: Which notebook does `week4_approx/dqn_atari_pytorch.ipynb` derive from? `week04_approx_rl/homework_pytorch_main.ipynb` or `week04_approx_rl/homework_pytorch_debug.ipynb`?

Comment (yhn112) [reply week4_approx/dqn_atari_pytorch.ipynb:1]: `homework_pytorch_main.ipynb` obviously

**Comment by dniku** [comment]
Could you port the latest updates from `spring20` to this branch? We'll merge it afterwards.

File: week5_policy_based/practice_reinforce_pytorch.ipynb [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+    "\"\"\".format(\"./videos/\" + video_names[-1]))  # this may or may not be the _last_ video. Try other indices"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from submit import submit_cartpole_pytorch\n",
```
Comment (dniku) [inline week5_policy_based/practice_reinforce_pytorch.ipynb:399]: It's not critical, but it would be nice to unify submission functions for PyTorch and TF.

Comment (yhn112) [reply week5_policy_based/practice_reinforce_pytorch.ipynb:399]: Sure

File: week1_intro/primer/recap_pytorch.ipynb [hunk 3]
```diff
@@ -0,0 +1,1282 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "if 'google.colab' in sys.modules:\n",
+    "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/74eff0033306bdcc24b799d6b311baa408245257/week04_%5Brecap%5D_deep_learning/notmnist.py -O notmnist.py\n",
+    "    !pip3 install torch==1.0.0 torchvision\n",
```
[the remark this answers is not in the document]
Comment (dniku) [reply week1_intro/primer/recap_pytorch.ipynb]: That's https://github.com/yandexdataschool/Practical_RL/issues/306 btw.

**Pull request merged**